envi/tests/test_arch_arm.py

```diff
@@ -140,38 +133,12 @@
         (REV_ALL_ARM, '8745d3e1', 0x4560, 'bics r4, r3, r7, lsl #11', 0, ()),
         (REV_ALL_ARM, '8745e3e1', 0x4560, 'mvn r4, r7, lsl #11', 0, ()),
         (REV_ALL_ARM, '8745f3e1', 0x4560, 'mvns r4, r7, lsl #11', 0, ()),
-        #(REV_ALL_ARM, '974523e0', 0x4560, 'mla r3, r7, r5, r4', 0, ()), #Emu: Unsupported instruction
```

Comment:
Expecting that they are not implimented. To be addressed later... Once full disasm coverage has been reached.